README.md

```diff
@@ -1,13 +1,12 @@
 # cran-downloads
 
-A shiny app to visualise CRAN downloads with data from the [cranlogs](https://github.com/metacran/cranlogs) package.
+A shiny app to visualise CRAN downloads with data from the [cranlogs](https://r-hub.github.io/cranlogs/) package.
 
 Live app at <https://hadley.shinyapps.io/cran-downloads/>
 
 ## To use
 
 ```R
-# install.packages("devtools")
-devtools::install_github("metacran/cranlogs")
-devtools::install_github("rstudio/shiny")
+install.packages("cranlogs")
+install.packages("shiny")
 ```
```

Comment:
unless there's a reason for installing `shiny` dev version?

Reply:
There probably was when the read me was written 😉

Reply:
that's what I assumed :grin: